src/runtime/gc_conservative.go

```diff
@@ -220,6 +220,10 @@ func alloc(size uintptr) unsafe.Pointer {
 		return unsafe.Pointer(&zeroSizedAlloc)
 	}
 
+	if interrupt.In() {
```

Comment:
Please let me know if you can think of a better name than `In`. This is the best I could come up with (`interrupt.InInterrupt` adds stutter, for example).

Reply:
How about `interrupt.IsBeingServiced()`

Reply:
Seems a bit long to me, but is descriptive.
I'm fine either way, I can update the PR if you prefer this name.